src/py42/services/detectionlists/user_profile.py

```diff
@@ -171,7 +172,14 @@ def add_cloud_alias(self, user_id, alias):
             u"cloudUsernames": [alias],
         }
         uri = self._make_uri(u"/addcloudusernames")
-        return self._connection.post(uri, json=data)
+        try:
+            return self._connection.post(uri, json=data)
+        except Py42BadRequestError as err:
+            if str(err.response.text).__contains__(
```

Comment:
Instead of calling this magic method `__contains__`, you can just do: 

```
if "Cloud usernames must be less than or equal to 2" in err.response.text:
```

Reply:
changed.